analyses/snv-callers/util/plot_functions.R

```diff
@@ -34,18 +34,23 @@ base_change_plot <- function(vaf_df, exp_strategy = "BOTH", filter_cutoff = 0) {
 
   # Count the number of each type of base change
   base_count_df <- vaf_df %>%
-    dplyr::mutate(change = as.factor(change),
+
+    dplyr::mutate(
+      change = as.factor(change),
       # Change factor level order so ins and del are at the end
-      change = factor(change, c(grep("ins|del", levels(change), value = TRUE, invert = TRUE), "ins", "del"))) %>% 
+      change = factor(change, c(grep("ins|del", levels(change), value = TRUE, invert = TRUE), "ins", "del"))
```

Comment:
Is this where the long_change snuck in? If so, move it to the back too?

Easier way to to this is with `forcats`
`forcats::fct_relevel(change, "ins", "del", "long_change", after = Inf)`

Reply:
Cool beans. I didn't like how messy my method was but wasn't sure of a cleaner alternative.